aliases/available/git.aliases.bash

```diff
@@ -12,9 +12,13 @@ alias gap='git add -p'
 # branch
 alias gb='git branch'
 alias gbD='git branch -D'
-alias gba='git branch -a'
 alias gbd='git branch -d'
-alias gbm='git branch -m'
+alias gba='git branch --all'
+alias gbr='git branch --remotes'
+alias gbl='git branch --list'
+alias gbla='git branch --list --all'
+alias gblr='git branch --list --remotes'
+alias gbm='git branch --move'
 alias gbt='git branch --track'
 alias gdel='git branch -D'
```

Comment:
Would you be so kind as to sort these?  gba above gbD, gbr above gbt gbdel below gbd

Reply:
Done.